apps/meteor/tests/end-to-end/api/settings.ts

```diff
@@ -100,6 +100,20 @@ describe('[Settings]', () => {
 				})
 				.end(done);
 		});
+		it('should return the default values of the settings when includeDefaults is true', async () => {
```

Comment:
I'm wondering if it'd be good to add a test to cover the case where this param is not provided (or adapt the old ones) so that we can assure `packageValue` is not being returned by default

Reply:
Actually, was thinking on that, but at the end didn't feel any need for it. There was no way for that to be possible, so i ended up with just the positive test.